app.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+
+hamburger.addEventListener("click", () => {
+    hamburger.classList.toggle("active");
+    navMenu.classList.toggle("active");     
+})
+document.querySelectorAll(".nav-link").forEach(n => n. addEventListener("click", () =>{
+    hamburger.classList.remove("active");
+    hamburger.classList.remove("active");
+
+}))
```

Comment:
Great job implementing the functionality of the hamburger menu but for now it is not part of the requirements so please delete it and save it for when you need it. Keep rocking 💪